### ISSUE TITLE
Timeline

### ISSUE THREAD

**Issue by dorothealint** [dorothealint/Rick-and-Morty#20]
@quantum-satire  How static is your timeline on the webpage?  Can it be added to?  I just put in a bunch of years on the episodes...I have season 3 to go yet.  They will be at the bottom in a \<trailer>\<name>\</name>\<date>\</date>\</trailer> element inside the body of the documents.  I'm almost done with them and I will push them.  But I was thinking we could do a quick Javascript to display the episode name when you run your mouse over the dot on the timeline just to have some fancier output.

**Comment by helvitiis** [comment]
@dorothealint What's on there is what I have so far--I was working on the transcripts view more-so than the timeline, but I plan on getting to the timeline tonight and tomorrow. I am thinking we may need to do something entirely static because of the time limit to Friday, and then shift back to the transcripts view. What do you think? Also, if you push what you have, I can go ahead and get that onto the server after styling it and tucking it away where it needs to be :)

**Comment by dorothealint** [comment]
@quantum-satire How come my changes to script files are not appearing in exide???  It did it this morning when we moved all the scripts!  I pushed what I need to git hub.

**Comment by dorothealint** [comment]
I think they only change when you push them because I don't have access to even change those script files by hand!

**Comment by helvitiis** [comment]
@dorothealint It may be because I pushed all of the files to eXide and it becomes angry when someone tries to mess with files that were written by someone else--I'll pull in the changes from the repo and push them onto eXide. Also, I'll be heading to 131 around noon today and staying until 5-ish if you want to stop by and we can work on the project!

**Comment by dorothealint** [comment]
Yes!  I was just going to message you and ask if you were going in!  I will definitely come in.

**Comment by helvitiis** [comment]
@dorothealint Great! See you then!
